src/notifications/notification.ts

```diff
@@ -137,19 +138,27 @@ export default class Notification implements Comparable<Notification> {
     return 0;
   }
 
-  public toMDString(): string {
+  public toMDString(limit?: number): string {
     const titleText = this.title.link
       ? `[**${this.title.text}**](${this.title.link})`
       : this.title.text;
 
+    let mdStr;
+
     if (this.author.text) {
       const authorText = this.author.link
         ? `[${this.author.text}](${this.author.link})`
         : this.author.text;
 
-      return `New **${this.game.label}** update - ${authorText}:\n\n${titleText}\n\n${this.content}`;
+      const contentText = this.content ? `\n\n${this.content}` : '';
+      mdStr = `New **${this.game.label}** update - ${authorText}:\n\n${titleText}${contentText}`;
+    } else {
+      const contentText = this.content ? `\n\n${this.content}` : '';
+      mdStr = `New **${this.game.label}** update:\n\n${titleText}${contentText}`;
```

Comment:
you can just set up the author text including the dash inside the if and use it anyway, similar to the content